rector.php

```diff
@@ -15,7 +20,18 @@
 
     $parameters->set(Option::SKIP, [
         CountOnNullRector::class,
+
+        // for PHP 8
+        RemoveUnusedPromotedPropertyRector::class,
+
+        // buggy when remove private property, but filled by construct in the middle of parameter
+        RemoveUnusedPrivatePropertyRector::class,
```

Comment:
improvements are coming to rector:

- https://github.com/rectorphp/rector-src/pull/1212
- https://github.com/rectorphp/rector-src/pull/1214